trl/trainer/ppo_trainer.py

```diff
@@ -1022,6 +1023,10 @@ def train_minibatch(
             train_stats (dict[str, `torch.Tensor`]):
                 Dictionary of training statistics
         """
+        # This is needed to avoid DeepSpeed throwing `backward pass is invalid for module in evaluation mode`
+        if self.accelerator.state.deepspeed_plugin.zero_stage == 3:
+            self.model.train()
```

Comment:
Would it make sense to put the model in train mode for all accelerators (i.e. not just DeepSpeed)?